nntrainer/graph/network_graph.cpp

```diff
@@ -422,12 +421,14 @@ int NetworkGraph::addLossLayer(const LossType loss_type) {
     std::dynamic_pointer_cast<LossLayer>(layer)->setLoss(updated_loss_type);
   NN_RETURN_STATUS();
 
+  ensureName(layer);
   if (last_node.layer->getType() == TimeDistLayer::type) {
+    std::string unit_str = layer->getName();
+    layer->setName(unit_str + "_unit");
```

Comment:
same

Reply:
well.. I guess, layer->getName() is unique.. then this is unique. And the uniqueness is checked already.